portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Gallery of Favorites</h1>
+      <table>
```

Comment:
The table element should only be used for tabular data. This site does a great job of describing why:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics#When_should_you_NOT_use_HTML_tables

Reply:
Changed this to use Tamas' suggestion.